README.md

```diff
@@ -119,6 +119,9 @@ A system that monitors Mento Governance events on-chain and sends notifications
    # Get it from the [QuickNode dashboard](https://dashboard.quicknode.com/api-keys)
    quicknode_api_key    = "<quicknode-api-key>"
 
+   # Get it via `echo "\nquicknode_security_token = \"$(gcloud secrets versions access latest --secret quicknode-security-token)\"" >> terraform.tfvars`
```

Comment:
There are a few references to `quickalerts` left in the repo, would you mind replacing those? for example here in L118

Reply:
done!